metadata.yaml

```diff
@@ -44,3 +44,4 @@ requires:
   agent:
     interface: jenkins_agent_v0
     optional: true
+    limit: 2
```

Comment:
Why 2?

Reply:
There is a use-case where a k8s agent and a machine agent would connect to the same server. Both k8s agents and machine agents can scale by adding units. I didn't quite see other reasons to be able to add more agents through the relation, other than the 2 metioned :).

Reply:
Thanks for the clarification :)

Reply:
What about agents with different OS? E.g., one with 22.04 and another with 20.04? And agents with certain software pre-installed?